features/promotion/applying_catalog_promotions/not_reapplying_catalog_promotions_on_variants_once_products_taxon_changes.feature

```diff
@@ -0,0 +1,34 @@
+@applying_catalog_promotions
+Feature: Not reapplying catalog promotions on variants once the product’s taxon changes
+    In order to have proper discounts in product catalog
+    As a Store Owner
+    I do not want to have discounts reapplied on variants once the product’s taxon changes if the catalog promotion criteria are not met
+
+    Background:
+        Given the store operates on a channel named "Web-US" with hostname "web-us"
+        And the store classifies its products as "Clothes", "Shirts" and "Dishes"
+        And the store has a "T-Shirt" configurable product
+        And this product belongs to "Clothes"
+        And this product has "PHP T-Shirt" variant priced at "$100.00"
+        And it is "2022-01-01" now
+        And there is a catalog promotion "Winter sale" available in "Web-US" channel that reduces price by "30%" and applies on "Clothes" taxon
+        And this catalog promotion operates between "2021-12-20" and "2021-12-30"
+        And there is another catalog promotion "Spring sale" available in "Web-US" channel that reduces price by "25%" and applies on "Shirts" taxon
+        And this catalog promotion operates between "2022-04-01" and "2022-05-01"
+        And there is another catalog promotion "Surprise sale" available in "Web-US" channel that reduces price by "90%" and applies on "Dishes" taxon
+        And this catalog promotion is disabled
+        And I am logged in as an administrator
+
+    @ui
```

Comment:
Why these scenarios are only in `UI` context?

Reply:
After some investigation, it seems that at that moment there is no possibility to manage the `products taxons` by API

![image](https://user-images.githubusercontent.com/40125720/150755955-964db6c2-7d58-4b61-8cc0-5f48d01da569.png)

Reply:
So, can you tell me how it will behave in API? :)
Maybe we should add some basic implementation o taxons in API WDYT cc @GSadee

Reply:
IMO not in this PR as it won't be a simple one, we need to rethink how to manage these entities in API `Product` <> `ProductTaxon` <> `Taxon`